### ISSUE TITLE
Default Keystore

### ISSUE THREAD

**Issue by BenjamenMeyer** [karakun/OpenWebStart#434]
I am trying to run OpenWebStart to access some JNLP resources; however, I can't seem to figure out how to get it to detect the default keystore. I've run the itw-settings and itweb-settings tools provided by OpenWebStart and the IcedTea JRE/netx (icedtea-netx on Ubuntu 20.04); neither seem to have issues and both find the system CA Certs as well as the two I specifically loaded for the JNPL services.

Also at least the Linux directions in the dialog on wrong.

Please advise.

![image](https://user-images.githubusercontent.com/1074110/124659677-38c56d00-de73-11eb-846f-d99e2ec3a4df.png)

OS: Ubuntu 20.04 x86-64
Desktop: KDE
JVMs: OpenJDK11, IcedTea
OpenWebStart version: 1.4.0 (Deb file)

**Comment by sclassen** [comment]
This dialog is not from OWS but from the application you are running.
We have no information about how this application is trying to find a keystore.
Best is to contact the developers of the application to have them help configuring the app with OWS.

**Comment by BenjamenMeyer** [comment]
> This dialog is not from OWS but from the application you are running.
> We have no information about how this application is trying to find a keystore.
> Best is to contact the developers of the application to have them help configuring the app with OWS.

Thanks.

Unfortunately in my case the product is an EOL'd hardware product that we have in a lab which won't get replaced any time soon.  It would be nice if there was some way to mend these situations.

**Comment by BenjamenMeyer** [comment]
> If you have an Oracle JVM installed you can use the following steps to open the control panel
> https://java.com/en/download/help/enable_console_linux.html
> 
> Since we do not know how the application is implemented it is best to try running it with different JVMs (incl Oracle) to make sure it is not a JVM specific behavior which prevents the application from finding the certificates.

Unfortunately I'm not sure that's a solution unless you want to keep really really old versions of Java installed. I suspect that it would work if I found an appropriately old version of Java (like JVM 6) where javaws is already included thus making this project unnecessary.

That said, we did manage to find a newer firmware for the devices that works with the Iced Tea provided javaws (at least for now). It likely won't ever see an upgrade again so hopefully things keep working.

Since this is no longer an issue for me, I'm going to close it for now. Someone else can open if they find this is an issue for them.
